types/test/isSet.ts

```diff
@@ -0,0 +1,16 @@
+import * as RA from 'ramda-adjunct';
+
+RA.isSet(+0); // $ExpectType boolean
+RA.isSet(null); // $ExpectType boolean
+RA.isSet(true); // $ExpectType boolean
+RA.isSet(NaN); // $ExpectType boolean
+RA.isSet({}); // $ExpectType boolean
+RA.isSet([]); // $ExpectType boolean
+RA.isSet(() => {}); // $ExpectType boolean
+RA.isSet('string'); // $ExpectType boolean
+RA.isSet(1); // $ExpectType boolean
+RA.isSet(Object()); // $ExpectType boolean
+RA.isSet(Object(5)); // $ExpectType boolean
+RA.isSet(new Map()); // $ExpectType boolean
+RA.isSet(new Set()); // $ExpectType boolean
```

Comment:
Move doing assertions about set on top. We always want to have positive stuff on top, and then continue with all the negative ;]

Reply:
I think I have done so now